samples/openai_chatgpt/program.py

```diff
@@ -51,13 +47,11 @@ def on_slack_slash_command(event):
     ]
     msgs = [
         {"role": "system", "content": contents[0]},
-        {"role": "user", "content": contents[1]},
-        {"role": "user", "content": event.data["text"]},
+        {"role": "user", "content": text if text else contents[1]},
     ]
 
     resp = chatgpt_client.chat.completions.create(model=MODEL, messages=msgs)
 
-    id = event.data["user_id"]
     for choice in resp.choices:
-        slack_client.chat_postMessage(channel=id, text=choice.message.content)
-    slack_client.chat_postMessage(channel=id, text=f"Usage: `{str(resp.usage)}`")
+        print(choice.message.content)
+    print(f"Usage: `{str(resp.usage)}`")
```

Comment:
I think f-strings automatically use `str()` (or `repl()` if you specify `!r`), so no need to call it explicitly